bundles/binding/org.openhab.binding.lgtv/src/main/java/org/openhab/binding/lgtv/internal/LgtvBinding.java

```diff
@@ -204,8 +204,6 @@ private String convertOpenHabCommandToDeviceCommand(Command command, String cmdT
         } else if (command instanceof DecimalType) {
             deviceCmd = String.format(cmdTemplate, ((DecimalType) command).intValue());
 
-        } else if (command instanceof PercentType) {
```

Comment:
Explain or revert this change.

Reply:
`PercentType` is a subclass of `DecimalType`, and the test for `command instanceof PercentType` is only evaluated after it has been found to be not an instance of `DecimalType` (and consequently not an instance of any subclass of `DecimalType`). The test will always evaluate to false, so there is dead code following. The change I made removes the redundant check and the dead code.